common/src/main/java/com/codeforcommunity/enums/AssignedPersonTitle.java

```diff
@@ -0,0 +1,30 @@
+package com.codeforcommunity.enums;
+
+public enum AssignedPersonTitle {
+  LIBRARIAN("LIBRARIAN"),
+  SCHOOL_SECRETARY("SCHOOL_SECRETARY"),
+  CLASSROOM_TEACHER("CLASSROOM_TEACHER"),
+  APPRENTICE("APPRENTICE"),
+  PCV("PCV"),
+  OTHER("OTHER");
+
+  private String name;
+
+  AssignedPersonTitle(String name) {
+    this.name = name;
+  }
+
+  public String getName() {
+    return name;
+  }
+
+  public static AssignedPersonTitle from(String name) {
+    for (AssignedPersonTitle title : AssignedPersonTitle.values()) {
```

Comment:
nit: I don't think it matters too much, but maybe compile each of the names in a set in a static block or something and check presence there?

also nit: maybe uppercase or do `String.equalsIgnoreCase` for comparison?